e2e-tests/gef/cypress/e2e/automatic-cover.spec.js

```diff
@@ -38,18 +38,18 @@ context('Automatic Cover Page', () => {
       automaticCover.continueButton().click();
       automaticCover.errorSummary();
       automaticCover.fieldError();
-      automaticCover.automaticCoverTerm().its('length').should('be.gt', 0); // greater than
+      automaticCover.automaticCoverTerm(12).its('length').should('be.gt', 0); // greater than
     });
 
     it('removes error message from field if a radio button has been selected', () => {
       automaticCover.trueRadioButton().first().click();
       automaticCover.continueButton().click();
-      automaticCover.automaticCoverTerm().eq(0).siblings('[data-cy="automatic-cover-error"]').should('not.exist');
-      automaticCover.automaticCoverTerm().eq(1).siblings('[data-cy="automatic-cover-error"]'); // second term
+      automaticCover.automaticCoverTerm(12).eq(0).siblings('[data-cy="automatic-cover-error"]').should('not.exist');
+      automaticCover.automaticCoverTerm(12).eq(1).siblings('[data-cy="automatic-cover-error"]'); // second term
```

Comment:
```suggestion
      automaticCover.automaticCoverTerm(12).siblings('[data-cy="automatic-cover-error"]').should('not.exist');
      automaticCover.automaticCoverTerm(13).siblings('[data-cy="automatic-cover-error"]').should('exist'); 
```
I think this text is trying to get EC 12 and EC 13, and check that the error shows for EC 13 but not EC 12